src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx

```diff
@@ -46,12 +48,14 @@ const uiSchema = {
     ...phoneConfig,
     'ui:errorMessages': {
       ...phoneConfig['ui:errorMessages'],
-      pattern:
-        'Please enter a valid 10-digit phone number (with or without dashes)',
+      pattern: 'Please enter a 10-digit phone number (with or without dashes)',
     },
   },
   email: {
     'ui:title': 'Your email address',
+    'ui:errorMessages': {
```

Comment:
Huh, it's weird that we're not currently using `emailConfig` here. I think we should fix that. We can set `email: emailConfig,` and that should be good.

Reply:
Fixed